layouts/partials/scripts/mermaid.html

```diff
@@ -10,15 +10,19 @@
 {{ end -}}
 
 <script type="module" async>
-import mermaid from "{{ $cdnurl }}";
+import mermaid from "{{- $cdnurl -}}";
```

Comment:
I don't believe that this edit has any effect since there is no whitespace to trim outside of the double braces since they are immediately (each) next to a double quote.

Reply:
You are right. Reverted to original version.